docs-public/snippets.js

```diff
@@ -392,8 +416,24 @@ function bootstrapReact(options) {
     .join('\n');
 }
 
+function bootstrapVue(options) {
+    // in vue we extract the imports from html
+    var code = options.example.code;
+    var html = options.example.html;
+    var directives = usedModules(html);
+    var imports = moduleImports(html, directives.filter(function(item) { return !/react|angular/gi.test(item.module); }));
+    return [].concat([
+        "import Vue from 'vue';"
+    ])
+    .concat(imports)
+    .concat(code)
+    .filter(Boolean)
+    .join('\n');
+}
+
 function bootstrapAngular(options) {
-    var code = wrapAngularTemplate(options.code);
+    var source = options.example.ts;
+    var code = wrapAngularTemplate(source);
```

Comment:
This line breaks inline angular examples like https://www.telerik.com/kendo-angular-ui-develop/components/gauges/